android/expoview/src/main/java/host/exp/exponent/notifications/ActionObject.kt

```diff
@@ -0,0 +1,38 @@
+package host.exp.exponent.notifications
+
+import com.raizlabs.android.dbflow.annotation.Column
+import com.raizlabs.android.dbflow.annotation.PrimaryKey
+import com.raizlabs.android.dbflow.annotation.Table
+import com.raizlabs.android.dbflow.structure.BaseModel
+
+@Table(database = ActionDatabase::class)
+class ActionObject : BaseModel {
+  @Column var categoryId: String? = null
+  @PrimaryKey @Column var actionId: String? = null
+  @Column var buttonTitle: String? = null
+  @Column var isDestructive: Boolean? = null
+  @Column var isAuthenticationRequired: Boolean? = null
+  @Column var submitButtonTitle: String? = null
+  @Column var placeholder: String? = null
+  @Column var isShouldShowTextInput: Boolean? = null
```

Comment:
As far as I can tell they defaulted to null which in sqlite is represented as an `INTEGER` column type that defaults to 0 which is the same thing that false would default to (just in case they didn't default to null).